### PR TITLE
use liboverdrop crate

### PR DESCRIPTION
**Pull request by rfairley**
Use the API provided by https://github.com/overdrop/liboverdrop-rs/pull/1.

### CODE REVIEW

File: src/config/inputs.rs [hunk 1]
```diff
@@ -40,17 +16,15 @@ pub(crate) struct ConfigInput {
 impl ConfigInput {
     /// Read config fragments and merge them into a single config.
     pub(crate) fn read_configs(
-        dirs: &[path::PathBuf],
+        dirs: &Vec<String>,
```
Comment (lucab) [inline src/config/inputs.rs]: * If consumer passes ownership to this: `Vec<String>`
* If consumer does not pass ownership to this: `&[String]`

Comment (rfairley) [reply src/config/inputs.rs]: Thanks, fixed!

**Comment by rfairley** [comment]
Accidentally pushed https://github.com/rfairley/fedora-coreos-metrics-client/commit/63f903631d3a68cf0ac70fcdfbff1b69a51f55aa to master, closing this PR. This was a simple change to adapt to the changes in the [liboverdrop](https://github.com/overdrop/liboverdrop-rs) master - so I think it is OK not to revert it. A follow-up can be done soon to use the `liboverdrop` crate once published.

**Comment by lucab** [comment]
I guess this means we now need to release and fedora-package `liboverdrop` before being able to release and package this. Timewise, do you think it's fine?

**Comment by rfairley** [comment]
Right, yes this does make packaging `liboverdrop` a blocker. The packaging for `liboverdrop` should be straightforward, but I don't want to rush work there for this. I'll revert this now - less harm done.

I did build an RPM ready go [here](https://github.com/rfairley/fedora-coreos-metrics-client/issues/3#issuecomment-499256969) for FCOS, but we may still want to add changes upstream (in this repo) before packaging this. Shouldn't block more changes being added based on my accidental push.

**Comment by rfairley** [comment]
Superseded by: #10

**Pull request closed without merging**